everviz/data/load_csv/_get_data.py

```diff
@@ -0,0 +1,9 @@
+import pandas as pd
+from webviz_config.common_cache import CACHE
+from webviz_config.webviz_store import webvizstore
+
+
+@CACHE.memoize(timeout=CACHE.TIMEOUT)
+@webvizstore
+def get_data(data_path) -> pd.DataFrame:
+    return pd.read_csv(data_path, delimiter=";")
```

Comment:
I am confused about the naming of this file. It is not clear for me why the file needs the `_` in front of the name. 

As far as I am aware, when you add an `_` in front of a file name it is indicating that is should not be imported outside the module it is defined in.